cve_bin_tool/version.py

```diff
@@ -31,12 +31,10 @@ def check_latest_version():
             else:
                 # TODO In future mark me with some color ( prefer yellow or red )
                 LOGGER.info(
-                    textwrap.dedent(
-                        f"""You are running version {VERSION} of {name} but the latest is {pypi_version}!
-                    Alert: We recommend using the latest stable release.
-                    """
-                    )
+                    f"You are running version {VERSION} of {name} but the latest PyPI Version is {pypi_version}."
                 )
+                if VERSION < pypi_version:
```

Comment:
You need to parse version before comparing. string will comparison will give false positives in some cases.

Reply:
I have tested this against many versions but none of them had given false positive.

Reply:
It'll likely work as long as our versions are x.x instead of x.x.x, and I wasn't planning to change that (I dropped the last number when we hit 1.0 beause I'm not seeing us releasing often enough to need that level of release revision) but let's just do it right and use parse version and save us a potentially confusing bug later.